modules/client-configuration/pages/client-upgrades-sp-migration.adoc

```diff
@@ -7,7 +7,6 @@ For example, you can migrate {sle}{nbsp}15{nbsp}SP1 to {sle}{nbsp}15{nbsp}SP2.
 You can also migrate openSUSE Leap to a later minor version or to the corresponding SLE SP level, for example:
 
 * openSUSE Leap 15.1 to 15.2, or
-* openSUSE Leap 15.1 to {sle} 15 SP1, or
 * openSUSE Leap 15.2 to {sle} 15 SP2
```

Comment:
I am not sure if SP migration from Leap 15.1 to SLE15SP1 is still supported or not, as SLE15SP1 is not EoL (but LTSS). Please check with @paususe

Reply:
Yes, this is supported.

![image](https://user-images.githubusercontent.com/50680111/115232294-d338ce80-a116-11eb-93b6-99e889a3424f.png)